packages/components/src/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactElement, ReactNode } from "react";
+import styles from "./RadioGroup.css";
+
+interface RadioGroupProps {
+  readonly children: ReactElement | ReactElement[];
+  readonly value: string;
+  onChange(newValue: string): void;
+  readonly name: string;
+}
+
+export function RadioGroup({
+  children,
+  value,
+  onChange,
+  name,
```

Comment:
If you're not submitting this through a form there isn't really need to have a human readable name. Might be nice to make name optional in the interface and then default it to a UUID.

```suggestion
  name = uuid.v1(),
```